src/libs/antares/study/area/area.cpp

```diff
@@ -36,6 +36,8 @@
 #include "constants.h"
 #include "ui.h"
 #include "scratchpad.h"
+#include "antares/study/parts/parts.h"
+#include "antares/study/parts/load/prepro.h"
```

Comment:
This should be included in parts.h

Reply:
It's complicated. Including parts.h and having prepro included by parts.h create some cyclical inclusion.